packages/roosterjs-react-command-bar/lib/components/RoosterCommandBar.tsx

```diff
@@ -43,24 +35,25 @@ export default class RoosterCommandBar extends React.Component<RoosterCommandBar
         return (
             <div className={css("rooster-command-bar", className)}>
                 <CommandBar
-                    // TODO-1 update experimental OfficeFabric CommandBar
-                    /* overflowItemProps={{ menuClassName: calloutClassName, onAfterMenuDismiss: calloutOnDismiss, } as IOverflowItemProps} */
-                    className={css("command-bar")}
+                    className={"command-bar"}
                     items={this._buttonKeys
-                        .map(key => this._getMenuItem(OOB_COMMAND_BAR_ITEM_MAP[key]))
+                        .map(key => this._getMenuItem(OutOfBoxCommandBarItemMap[key]))
                         .filter(menuItem => !!menuItem)}
                 />
+                <input type="file" ref={this._fileInputOnRef} accept="image/*" style={DisplayNoneStyle} onChange={this._fileInputOnChange} />
```

Comment:
nit: inline?

Reply:
I mean inline ref btw, dunno why GitHub doesnt highlight the selection :(

Reply:
it's less performant if you inline, the delegate is created each time render is called